util/Assert.c

```diff
@@ -29,6 +33,24 @@ void Assert_failure(const char* format, ...)
     va_start(args, format);
     vfprintf(stderr, format, args);
     fflush(stderr);
+
+#ifdef __GLIBC__
+    void *array[20];
+    size_t size;
+    char **strings;
+    size_t i;
+
+    size = backtrace (array, 20);
+    strings = backtrace_symbols (array, size);
+
+    fprintf(stderr, "Backtrace (%zd frames):\n", size);
+    for (i = 0; i < size; i++) {
```

Comment:
Really? I always use next line, but I especially looked at cjdns code and found the curly on the same line.